Test/allocated1/dafny0/Basics.dfy.expect

```diff
@@ -9,14 +9,14 @@ Basics.dfy(148,15): Error: assertion might not hold
 Basics.dfy(167,9): Error: when left-hand sides a[b[18]] and a[4] refer to the same location, they must be assigned the same value
 Basics.dfy(183,9): Error: when left-hand sides x.f and y.f refer to the same location, they must be assigned the same value
 Basics.dfy(195,18): Error: assertion might not hold
-Basics.dfy(197,9): Error: assignment may update an object not in the enclosing context's modifies clause
 Basics.dfy(197,9): Error: target object may be null
+Basics.dfy(197,9): Error: assignment may update an object not in the enclosing context's modifies clause
```

Comment:
Do you know why these positions changed?

Reply:
My guess, though I haven't confirmed this, is that these are sorted by position at some point, but that process wouldn't be stable with respect to multiple errors on the same line. By changing the structure of the objects in that collection, their order could change, as well.